css/style.css

```diff
@@ -0,0 +1,309 @@
+    width: 100%;
+    margin-left: auto;
+    margin-right: auto;
+
+}
+
+.skl {
+    width: 18%;
+    border: 5px double;
+    color: antiquewhite;
+    font-size: 30px;
+    text-align: center;
+    display: inline-block;
+    padding: 16px;
+    margin-bottom: 0.5rem;
+    margin-left: 70px;
+    
+}
+
+/* leave a message section */
+form{
+    width: 25em;
+    max-width: 90%;
+    margin: 0 auto;
+    font-family: 'Blanka', sans-serif;
+    font-size: 15px;
+    color: white;
+    border-radius: 5px;
+    background-color: #222222;
+    padding: 20px;
+
+}  
+     
+   
+input[type=text], select {
+    width: 100%;
+    padding: 12px 20px;
+    margin: 5px 0;
+    display: inline-block;
+    border: 2px solid #ccc;
+    border-radius: 4px;
+    box-sizing: border-box;
+
+}
+     
+input[type=email], select {
+    width: 100%;
+    padding: 12px 20px;
+    margin: 5px 0;
+    display: inline-block;
+    border: 2px solid #ccc;
+    border-radius: 4px;
+    box-sizing: border-box;
+
+}
+     
+textarea[type=message] {
+     width: 100%;
+    padding: 20px 20px;
+    margin: 5px 0;
+    border: 2px solid #ccc;
+    border-radius: 4px;
+    box-sizing: border-box;
+
+}
+     
+button{
+    font-family: 'Blanka', sans-serif;
+    font-size: 15px;
+    background-color: rgb(24, 24, 24);
+    border: none;
+    border-radius: 4px;
+    color: white;
+    padding: 16px 32px;
+    margin: 10px 2px;
+    cursor: pointer;    
+
+}
+   
+  
+/* footer */
+
+.footer-nav{
+    padding-top: 20px;
+    list-style: none;
+    display:block;
+    position: relative;
+   
+}
+
+.container {
+    padding: 1rem 0;
+    margin: 0 auto;
+    max-width: 1024px;
+    width: 90vw;
+    color: antiquewhite;
+    
+}
+
+.fa {
+    padding: 17px;
+    font-size: 30px;
+    width: 50px;
+    text-align: center;
+    text-decoration: none;
+    margin: 5px 5px;
+    border-radius: 70%;
+
+}
+
+.fa-linkedin {
+    background: #007bb5;
+    color: white;
+}
+
+.fa-github {
+    background: #dbd9d9;
+    color: rgb(4, 4, 4);
+}
+
+.fa-twitter {
+    background: #43bcf4;
+    color: rgb(255, 255, 255);
+}
+
+.fa-instagram {
+    background: #f09433; 
+    background: -moz-linear-gradient(45deg, #f09433 0%, #e6683c 25%, #dc2743 50%, #cc2366 75%, #bc1888 100%); 
+    background: -webkit-linear-gradient(45deg, #f09433 0%,#e6683c 25%,#dc2743 50%,#cc2366 75%,#bc1888 100%); 
+    background: linear-gradient(45deg, #f09433 0%,#e6683c 25%,#dc2743 50%,#cc2366 75%,#bc1888 100%); 
+}
+
+
+/* (phones, 600px and down) */
+@media only screen and (max-width: 600px) {
+    .image {
+        width: 250px;
+        margin: 0 1rem 1rem 0;
+    }
+}
+/* (tablets, 768px and up) */
+@media only screen and (min-width: 768px) {
+    .image {
+        width: 350px;
+        margin: 0 1rem 1rem 0;
+    }
+}
+/* (laptops/desktops, 992px and up) */
+@media only screen and (min-width: 992px) {
+    .image {
+        width: 400px;
+        margin: 0 1rem 1rem 0;
+    }
+}
```

Comment:
I would generally suggest sticking to just `min-width` or just `max-width` otherwise it can get a bit confusing. For example, here with your media queries you have blocks for:

*   0 to 600px width screens
*   768 to 992px width screens
*   screens larger than 992px

But what happens between 600px and 768px ? In that case it'll default to whatever else is in your stylesheet but that can make it easy to make mistakes for those screen sizes.

Reply:
So should I just leave it as

 @media only screen and (min-width: 768px) {
    .image {
        width: 350px;
        margin: 0 1rem 1rem 0;
    }
}

Reply:
I would suggest keeping everything as `min-width` for example:

```
@media only screen and (min-width: 1px) {
	/* styles for only phones, starting from 1 px screen width */
}


@media only screen and (min-width: 768px) {
    /* styles for only tablets, starting from screen width 768px and up */
}


@media only screen and (min-width: 992px) {
    /* styles for laptops/desktops, starting from screen width of 992px and up */
}
```

  
What you put in each section just depends on how you want the page to look on that size screen.  For example one thing I noticed while resizing my screen on Safari is that on phones (less than about 750 or so pixels) the about section starts to get covered by the "Experience" header. So maybe in that media query for phones (`min-width: 1px`) you could add some kind of margin or padding for that section. 

![](https://user-images.githubusercontent.com/9094098/211083163-2fb5777d-1973-4233-8492-eb98b255acf3.png)